Practicals/Bash_Practicals/1_IntroBash.md

```diff
@@ -182,7 +182,7 @@ This is an essential skill when logged into a High Performance Computer (HPC) or
 ### Important
 {:.no_toc}
 
-*Although we haven't directly discovered it yet, a Unix-based file system such as Ubuntu or MacOS is* **case-sensitive**, whilst **Windows is not**.
+*Although we haven't directly discovered it yet, most file systems used on Unix-based systems such as Ubuntu are* **case-sensitive**, whilst **Windows file systems are usually not**.
```

Comment:
There's a grammar issue here

Reply:
Oh hang on. Maybe it is correct

Reply:
This is correct. Both clauses are plural and conjugated as such.